docs/faq.md

```diff
@@ -53,8 +53,16 @@ The reasons why VirtualiZarr has been developed as separate project rather than
 
 ## What is the Development Status and Roadmap?
 
```

Comment:
Noting that any changes made to the readme should be reflected here. Should this information just be stated once and referenced in the alternate location?

Reply:
Yeah I mean normally I don't like to have the same text in two locations, but the readme is important as a landing page too. No strong opinions on this - I could just link to the docs from the readme.